substrate/frame/support/procedural/examples/section/main.rs

```diff
@@ -0,0 +1,29 @@
+use frame_support::pallet_macros::{import_section, pallet_section};
```

Comment:
So this failed to compile before this fix?

Could you please move this as test to `frame-support-test/test`. Maybe we should also put the sections into their own files to be even closer to reality.

Reply:
Yep, it complains that "cannot find value `entries` in this scope", just as described in the issue.